customer-wallet/src/containers/Transactions/TransactionInfo/index.js

```diff
@@ -0,0 +1,139 @@
+            convertFromBitsToUsd,
+            review,
+            editing
+        } = this.props
+
+        const amount = calculateTotalSpent(transaction, address)
+        const priceBits = convertSatoshiToBits(amount)
+        const confirmations = get(transaction, 'confirmations', 0)
+        const priceBitsFormatted = formatCurrency(priceBits)
+        const date = moment(transaction.received).calendar()
+        const isChlu = Boolean(review)
+        const reviewIsLoading = isChlu && review.loading
+        const reviewIsReady = isChlu && !review.loading
+        const reviewMultihash = isChlu ? review.multihash : null
+        const reviewIsUpdate = Boolean(review && review.last_version_multihash)
+        const reviewDate = reviewIsUpdate ? null : date
+        const chainExplorerLink = `${process.env.REACT_APP_BTC_EXPLORER_URL_TX}/${transaction.hash}`
+        const chluExplorerLink = isChlu ? `https://explorer.chlu.io/#/v/${reviewMultihash}` : null
+        
+        let status = {
+            primary: 'Confirmed',
+            icon: ValidIcon,
+            secondary: isChlu ? `Chlu data valid. ${confirmations} Blockchain confirmations` : 'Transaction confirmed'
+        }
+        if (isChlu && review && review.error) {
+            status = {
+                primary: 'Invalid',
+                icon: InvalidIcon,
+                secondary: 'Chlu data invalid'
+            }
+        } else if (confirmations < 6) {
+            status = {
+                primary: 'Unconfirmed',
+                icon: UnconfirmedIcon,
+                secondary: `${confirmations}/6 Blockchain Confirmations`
+            }
+        }
+
+        return <Card className={classes.card}>
+            <CardHeader
+                avatar={<Avatar><PaymentIcon/></Avatar>}
+                title={`Sent ${priceBitsFormatted} bits ${date}`}
+                subheader={`To ${transaction.addresses[1]}`}
+            />
+            <Divider/>
+            { !isChlu && <CardHeader
+                avatar={<Avatar><NotChluIcon/></Avatar>}
+                title={`Not a Chlu Transaction`}
+                subheader={`This Transaction does not contain valid Chlu data`}
+            />}
+            { reviewIsLoading && <CardHeader
+                avatar={<Avatar><LoadingIcon/></Avatar>}
+                title={`Fetching Review data from IPFS...`}
+                subheader={`This might take some time`}
+            />}
+            { reviewIsReady && <Review
+                convertFromBitsToUsd={convertFromBitsToUsd}
+                convertSatoshiToBits={convertSatoshiToBits}
+                transaction={transaction}
+                date={reviewDate}
+                review={review}
+                editing={editing}
+            /> }
+            <Divider/>
+            <CardContent>
+                <List dense disablePadding>
+                    <ListItem>
+                        <ListItemIcon><status.icon /></ListItemIcon>
```

Comment:
Can we use the color schemes for confirmations? These confirmations are important to btc people. They look out for such things.

= 0 red
< 6 yellow
>= 6 green